Bottom: 014baf6cffa98842a534c53aab78e90d2514512f
Top:    9af5d63adcbf66351842b0462d35ae615728c384
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-24 22:29:54 -0400

Refresh of efika-audio

---

diff --git a/sound/soc/fsl/efika-audio-fabric.c b/sound/soc/fsl/efika-audio-fabric.c
index 6e0d216..c3e9da0 100644
--- a/sound/soc/fsl/efika-audio-fabric.c
+++ b/sound/soc/fsl/efika-audio-fabric.c
@@ -195,11 +195,6 @@ static int efika_stac9766_probe(struct platform_device *pdev)
 	return 0;
 }
 
-static int __exit efika_stac9766_remove(struct of_device *op)
-{
-	return 0;
-}
-
 #ifdef CONFIG_PM
 
 static int efika_stac9766_suspend(struct platform_device *pdev,
